docs/shared-content-source/docs/modules/administration/pages/how-to-install-and-use-strimzi.adoc

```diff
@@ -0,0 +1,136 @@
+
+
+include::ROOT:partial$include.adoc[]
+
+This guide shows how to install Strimzi and create a Kafka cluster suitable for use with Cloudflow.
+
+Strimzi is a Kubernetes operator that simplifies the process of running Apache Kafka.
+
+== Prerequisites
+
+=== CLI Tools
+
+Make sure you have the following prerequisites installed before continuing:
+
+- Helm, version 3 or later
+- Kubectl 
+
+== Installing Strimzi
+
+In this guide, we will use Helm to install Strimzi.
+
+We are going to create Strimzi in the `cloudflow` namespace in this guide. Make sure that the namespace `cloudflow` exists before continuing. To create the `cloudflow` namespace, execute
+
+  kubectl create ns cloudflow
```

Comment:
is there a way for having those commands coming from a Makefile?
i.e.:
https://github.com/lightbend/cloudflow/blob/master/integration-tests/Makefile#L36

Reply:
If it is in the examples/doc-snippets dir. I'll check and see.

Reply:
This file is not in the examples/doc-snippets dir, but could be added there.